<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Informacion_base_de_datos</class>
 <widget class="QMainWindow" name="Informacion_base_de_datos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Temperatura:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>150</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Precipitación:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Dirección viento:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Velocidad viento:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>260</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Intensidad de Luz:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Hora- Minutos:</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>156</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="leftMargin">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>20</number>
     </property>
     <item>
      <widget class="QLineEdit" name="Hora"/>
     </item>
     <item>
      <widget class="QLineEdit" name="Minutos"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Recuperar_medida">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>430</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Recuperar medida</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>40</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Humedad:</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>331</width>
      <height>73</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Promedio</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="TempMin"/>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="TempProm"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Máximo</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="TempMax"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Minimo</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>331</width>
      <height>122</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="2">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Promedio</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>Minimo</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="DireccionVProm"/>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="DireccionVMax"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Máximo</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="DireccionVMin"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>331</width>
      <height>73</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="0" column="2">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>Promedio</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="VelocidadVMin"/>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="VelocidadVProm"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>Máximo</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="VelocidadVMax"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>Minimo</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>331</width>
      <height>73</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="2">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Promedio</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="HumedadMin"/>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="HumedadProm"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Máximo</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="HumedadMax"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Minimo</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>170</y>
      <width>331</width>
      <height>73</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="2">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Promedio</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="PrecipitacionMin"/>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="PrecipitacionProm"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Máximo</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="PrecipitacionMax"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Minimo</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>280</y>
      <width>331</width>
      <height>73</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="2">
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>Promedio</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="IntensidadLMin"/>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="IntensidadLProm"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>Máximo</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="IntensidadLMax"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_25">
       <property name="text">
        <string>Minimo</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="alerta">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>410</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QFrame{
background-color:green
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>380</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Led de alerta </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>746</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatos_Sensores">
    <property name="title">
     <string>Datos Sensores</string>
    </property>
   </widget>
   <addaction name="menuDatos_Sensores"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
